app/scripts/modules/core/src/presentation/icons/iconsByName.ts

```diff
@@ -25,18 +25,19 @@ import { ReactComponent as fnNew } from './vectors/fnNew.svg';
 import { ReactComponent as instances } from './vectors/instances.svg';
 import { ReactComponent as loadBalancer } from './vectors/loadBalancer.svg';
 import { ReactComponent as manualJudgement } from './vectors/manualJudgement.svg';
+import { ReactComponent as manualJudgementApproved } from './vectors/manualJudgementApproved.svg';
+import { ReactComponent as manualJudgementRejected } from './vectors/manualJudgementRejected.svg';
```

Comment:
@erikmunson a couple of new icons to differentiate the judgement outcomes

Reply:
oh cool, i'll try to work them in. right now the code is not setup to show a different icon based on constraint status but i think it'd be pretty trivial to do that.